server/models/index.js

```diff
@@ -0,0 +1,54 @@
+//both: name, email, password, rating, city
```

Comment:
Suggest file rename to User.js (use capital filenames for models) 

In index.js import all the individual models and exports them in an object. See wk 22 activity 22 for example.